Xamarin.Forms.Core/ListProxy.cs

```diff
@@ -26,8 +27,9 @@ internal sealed class ListProxy : IReadOnlyList<object>, IListProxy, INotifyColl
 
 		int _windowIndex;
 
-		internal ListProxy(IEnumerable enumerable, int windowSize = int.MaxValue)
+		internal ListProxy(BindableObject ItemView, IEnumerable enumerable, int windowSize = int.MaxValue)
```

Comment:
Can we change this signature from `internal ListProxy(BindableObject ItemView, IEnumerable enumerable, int windowSize = int.MaxValue)` to `internal ListProxy(IEnumerable enumerable, int windowSize = int.MaxValue, IDispatcher dispatcher = null)`? 

We don't need a full `BindableObject`, we just need the dispatcher. And if we have a dispatcher, we don't need the casts to `Element`.

And by making it optional, we don't have to update all the constructors in `ListProxyTests`. Those changes can just be reverted.

Reply:
Thank you! I made the adjustments.